src/apps/investments/client/projects/tasks.js

```diff
@@ -1,14 +1,15 @@
-import axios from 'axios'
+import { apiProxyAxios } from '../../../../client/components/Task/utils'
 
 import { getMetadataOptions } from '../../../../client/metadata'
 import { transformInvestmentProjectToListItem } from './transformers'
+import { validateTeamAdvisersAreUnique } from './validator'
 
 const handleError = (error) => Promise.reject(Error(error.response.data.detail))
 
 export const getProjects = ({ limit = 10, page, companyId, ...rest }) => {
   let offset = limit * (parseInt(page, 10) - 1) || 0
 
-  return axios
+  return apiProxyAxios
     .post('/api-proxy/v3/search/investment_project', {
```

Comment:
```suggestion
    .post('/v3/search/investment_project', {
```
I don't think `api-proxy` is needed here anymore

Reply:
I love this, lets clean up all of them, so I won't resolve this, but create a ticket for resolving all of these

Reply:
https://uktrade.atlassian.net/jira/software/projects/TET/boards/310?selectedIssue=TET-89 created to do all of them